samples/highcharts/annotations-advanced/controllable/demo.js

```diff
@@ -201,6 +201,106 @@ var chart = Highcharts.chart('container', {
                     }
                 }
             }]
+        }, {
+            type: 'ellipse',
+            point: {
+                x: 10,
+                y: 6,
+                xAxis: 0,
+                yAxis: 0
+            },
+            angle: 45,
+            rx: 40,
+            ry: 30,
+            controlPoints: [
+                {
+                    positioner: function (target) {
+                        var xy = Highcharts.Annotation.MockPoint.pointToPixels(
+                                target.points[0]
+                            ),
+                            attrs = target.getAttrsFromPoints();
```

Comment:
More popular variable name is `attribs` (the same in the method name)

Reply:
I would leave it as it is since there are a lot of `attrs` in the current code, e.g. `attrsMap` or `attrsFromOptions` etc.